package.json

```diff
@@ -18,6 +18,9 @@
     "start": "electron .",
     "lint": "eslint src/",
     "build": "yarn run setversion && electron-builder",
+    "indocker": "scripts/indocker.sh",
```

Comment:
can we call it something else or add punctuation? I'm probably the only one with the problem, but every time I've read this it's either "indicator" or something NSFW. "in-docker" would be fine imo.

Reply:
yeah, I'd be up for changing it - just used the value we'd been using previously

Reply:
yeah, I'd be up for changing it - just used the value we'd been using previously

Reply:
yeah, I'd be up for changing it - just used the value we'd been using previously

Reply:
yeah, I'd be up for changing it - just used the value we'd been using previously

Reply:
yeah, I'd be up for changing it - just used the value we'd been using previously